launchable/commands/subset.py

```diff
@@ -117,14 +117,21 @@ def separator(self, s: str):
             self._separator = s
 
         def test_path(self, path: TestPathLike):
+            def rel_base_path(path):
+                if isinstance(path, str):
```

Comment:
Almost good. But it's still a little bit hard to understand because there are two conditions. Let's gather them to one if statement.
```python3
return normpath(relpath(path, start=base_path)) if isinstance(path, str) and base_path else path
```
or
```python3
if isinstance(path, str) and base_path:
  return normpath(relpath(path, start=base_path))
else:
  teturn path
```